packages/wrangler/package.json

```diff
@@ -65,7 +65,8 @@
 		"start": "npm run bundle && cross-env NODE_OPTIONS=--enable-source-maps ./bin/wrangler.js",
 		"test": "npm run assert-git-version && jest --silent=false --verbose=true --forceExit",
 		"test-watch": "npm run test -- --runInBand --testTimeout=50000 --watch",
-		"test:ci": "npm run test -- --verbose=true --coverage"
+		"test:ci": "npm run test -- --verbose=true --coverage",
+		"test:smoke": "esbuild --bundle scripts/smoke-test.ts --outfile=scripts/smoke-test.mjs --platform=node --format=esm && zx scripts/smoke-test.mjs && rm scripts/smoke-test.mjs"
```

Comment:
I feel like we should not be polluting the `scripts` folder with generated files, even if they are gitignored. Can we put these in some `dist-...` folder that is gitignored?